lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingResult.cxx

```diff
@@ -541,7 +541,7 @@ CovarianceMatrix KrigingResult::getConditionalCovariance(const Point & point) co
   const Point data(point);
   // 1) compute \sigma_{x,x}
   LOGINFO("Compute interactions Sigma_xx");
-  const SquareMatrix sigmaXX(covarianceModel_(Point(inputDimension, 0.0)));
+  const CovarianceMatrix sigmaXX(covarianceModel_(Point(inputDimension, 0.0)));
```

Comment:
you might keep `SquareMatrix`

Reply:
Why? There is an other method where `sigmaXX` is defined as a `CovarianceMatrix`: https://github.com/openturns/openturns/blob/63f500d8a18ea09bc8d9fc3ed9db5efdbbc61147/lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingResult.cxx#L452

Reply:
yes. But this one is just an evaluation (should return square) whereas the second is a discretization (should return a covariance)

Reply:
OK. I've updated the commit and it now only contains your other correction.